app/v3/services/SubmitForeignPropertyBsasService.scala

```diff
@@ -40,33 +40,43 @@ class SubmitForeignPropertyBsasService @Inject()(connector: SubmitForeignPropert
       correlationId: String): Future[Either[ErrorWrapper, ResponseWrapper[Unit]]] = {
 
     val result = for {
-      desResponseWrapper <- EitherT(connector.submitForeignPropertyBsas(request)).leftMap(mapDesErrors(mappingDesToMtdError))
+      desResponseWrapper <- EitherT(connector.submitForeignPropertyBsas(request)).leftMap(mapDesErrors(errorMap))
```

Comment:
I believe in my endpoint PR I renamed `mapDesErrors`   to `mapDownstreamErrors` and the corresponding class to `DownstreamResponseMappingSupport', so think this will create a merge conflict if mine gets merged in first. Which naming convention is better do you think?

Reply:
Your rename is better. I won't touch it here as it might make more conflicts.